clippy_lints/src/doc/empty_docs.rs

```diff
@@ -0,0 +1,20 @@
+use super::EMPTY_DOCS;
+use clippy_utils::diagnostics::span_lint_and_help;
+use rustc_ast::Attribute;
+use rustc_lint::LateContext;
+use rustc_resolve::rustdoc::{attrs_to_doc_fragments, span_of_fragments};
+
+// TODO: Adjust the parameters as necessary
+pub(super) fn check(cx: &LateContext<'_>, attrs: &[Attribute]) {
+    let (fragments, _) = attrs_to_doc_fragments(attrs.iter().map(|attr| (attr, None)), true);
+    if let Some(span) = span_of_fragments(&fragments) {
```

Comment:
You already have this information in the function calling this one. Instead of re-computing it, pass the result of `span_of_fragments` directly or remove this whole file and emit the lint directly in the other.